boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -5,6 +5,74 @@ if(SEPARATE_DBG)
     set(CMAKE_LDR_PE_HELPER_STANDARD_LIBRARIES "-lgcc" CACHE STRING "Standard C Libraries")
 endif()
 
+if(NOT MSVC)
+###
+### For GCC
+###
+function(add_linker_script _target _linker_script_file)
+    get_filename_component(_file_full_path ${_linker_script_file} ABSOLUTE)
+    add_target_link_flags(${_target} "-Wl,-T,${_file_full_path}")
+
+    # Unfortunately LINK_DEPENDS is ignored in non-Makefile generators (for now...)
```

Comment:
And this is an issue because?

Reply:
That's just a remark because according to CMake documentation, the LINK_DEPENDS property is enabled only for the "make" generator, and not for Ninja (in our old CMake version we are using). Newer CMake versions support the LINK_DEPENDS also for Ninja, see https://cmake.org/cmake/help/latest/prop_tgt/LINK_DEPENDS.html .